lib/puppet/provider/cert_file/posix.rb

```diff
@@ -0,0 +1,63 @@
+require 'pp'
```

Comment:
I'm guessing this is left behind from tests?

Reply:
It is actually required for debug messages like [posix.rb#L16](https://github.com/rtib/puppet-openssl/blob/c4ab4c7ca731eb144cc03c1d1a6377946706c346/lib/puppet/provider/cert_file/posix.rb#L16)